app/src/main/java/com/duckduckgo/app/browser/BrowserTrackersAnimatorHelper.kt

```diff
@@ -187,17 +186,23 @@ class BrowserTrackersAnimatorHelper {
         }
     }
 
-    @SuppressLint("ObjectAnimatorBinding")
-    fun animateFadeOut(view: Any): ObjectAnimator {
-        return ObjectAnimator.ofFloat(view, "alpha", 1f, 0f).apply {
-            duration = DEFAULT_ANIMATION_DURATION
+    private fun animateFadeOut(view: View?): ObjectAnimator? {
```

Comment:
As a quick fix, that guard is ok.
But, what's the root cause of that null? are the animations being canceled when the view is destroyed? is there something we should be checking before that null gets there?

Reply:
So... in some cases what happens is the activity gets destroyed, if the dialog is being shown it then triggers `onDismissDialog` which is executed and causes the crash. 

I have further changed the code and I think it makes more sense now.